src/sql/parts/connection/common/providerConnectionInfo.ts

```diff
@@ -104,6 +104,23 @@ export class ProviderConnectionInfo implements data.ConnectionInfo {
 		this.options[name] = value;
 	}
 
+	/**
+	 * Returns the title of the connection
+	 */
+	public get title(): string {
+		let databaseName = this.databaseName ? this.databaseName : '<default>';
+		let userName = this.userName ? this.userName : 'Windows Authentication';
```

Comment:
I know it was like this in the old code but `<default>` and `Windows Authentication` should probably be localized